### DISCUSSION TITLE
Issue With Pillow 10.1.0

### DISCUSSION THREAD

**Discussion by crgrove** [obss/sahi#981] [General]
Not sure when it changed, but Image.save() no longer takes a "quality" parameter.  This breaks def _export_single_slice in slicing.py .

https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.save

**Comment by jokober** [comment]
Hi, are you actually getting an error? If yes what is the error?
As mentioned on the documentation you link keyword options are listed in the  in the [image format documentation](https://pillow.readthedocs.io/en/stable/handbook/image-file-formats.html). There you can still find quality. For other formats than jpeg this keyword should be ignored.

**Reply by crgrove** [reply]
When running the coco slice script (at least on Windows 11) it won't save the sliced images if the quality parameters is set in the call to Image.save().  You don't get an error, but the image output directory is empty when the script finishes.

**Comment by jokober** [comment]
This might be connected to  #973
